alerts/google-gke/http-gmp-server-errors-entire-workload.v1.json

```diff
@@ -0,0 +1,33 @@
+{
+  "displayName": "Server Error (5xx) is high. (GKE Workload: ${WORKLOAD_NAME})",
```

Comment:
Is it intended to have same displayName for two templates?

Reply:
Yes. We have server error and latency alerts for workloads per each different type of GKE loadbalancers and ingresses.